recipe/run_test.py

```diff
@@ -6,6 +6,9 @@
     from nose import run_exit
     from traits.etsconfig.api import ETSConfig
     import os
+    from subprocess import call
+    call("python setup.py build_ext --inplace",
```

Comment:
I think that you're right. Still,  shouldn't we try to find the real issue with py35?